packages/backend/src/queryBuilder.ts

```diff
@@ -79,6 +80,22 @@ const getDimensionFromId = (
                 };
         }
 
+        // At this point, we couldn't find the dimension with the given id in the explore
+        // it is possible that the explore is a joined table and is filtered by user_attributes
+        // So we check if the dimension exists in the unfiltered tables
+        if (
```

Comment:
_compileQuery already has the explore filtered out, that happens on the `getExplores` generic method at the beggining of `get chart results` .